pkg/repository/v1manifest/local_manifests.go

```diff
@@ -59,12 +64,16 @@ type FsManifests struct {
 // NewManifests creates a new FsManifests with local store at root.
 // There must exists the trusted root.json.
 func NewManifests(profile *localdata.Profile) *FsManifests {
-	return &FsManifests{profile: profile, cache: make(map[string]string)}
+	return &FsManifests{profile: profile, keys: NewKeyStore(), cache: make(map[string]string)}
```

Comment:
need a way to init keys by loading the trusted root locally?

Reply:
guess it's caused by this
```
➜  tiup git:(role) ✗ tt tivk
The component `tivk` is not installed; downloading from repository.
Failed to start component `tivk`
Error: Unknown role timestamp
➜  tiup git:(role) ✗
```

Reply:
I think I've addressed this in the last commit, when creating a new FsManifests, we now do some initialisation by reading in the root manifest and adding its keys to the key store.